nautobot/ipam/tests/test_models.py

```diff
@@ -990,3 +1030,8 @@ def test_vlan_validation(self):
             f'The assigned group belongs to a location that does not include location "{location.name}"',
             str(cm.exception),
         )
+
+
+class TestVRF(ModelTestCases.BaseModelTestCase):
+    model = VRF
+    # TODO(jathan): Add VRF model tests.
```

Comment:
Is this comment still relevant with the addition of this test case?

Reply:
You tell me :-) All the base class adds currently are some tests around natural-key and composite-key; I'm guessing that you had something beyond that in mind?

Reply:
I suppose we might want to exercise the new stuff, but I also this base test class didn't exist before...